### DISCUSSION TITLE
微信小程序能实现替换占位图功能吗

### DISCUSSION THREAD

**Discussion by jan0192** [Tencent/libpag#1872] [General] [answered]
微信小程序能实现替换占位图功能吗

**Comment by kevingpqi123** [comment, marked as answer]
使用完整版本是支持的，具体查看接入文档 https://pag.art/docs/sdk.html#%E5%BE%AE%E4%BF%A1%E5%B0%8F%E7%A8%8B%E5%BA%8F%E6%8E%A5%E5%85%A5
和 API 文档

**Comment by jan0192** [comment]
完整版是企业定制版吗


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Tencent/libpag"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月16日(星期四) 上午10:34
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Tencent/libpag] 微信小程序能实现替换占位图功能吗 (Discussion #1872)





 
使用完整版本是支持的，具体查看接入文档 https://pag.art/docs/sdk.html#%E5%BE%AE%E4%BF%A1%E5%B0%8F%E7%A8%8B%E5%BA%8F%E6%8E%A5%E5%85%A5
 和 API 文档

**Reply by kevingpqi123** [reply]
不是，看接入文档：https://pag.art/docs/sdk.html#%E5%BE%AE%E4%BF%A1%E5%B0%8F%E7%A8%8B%E5%BA%8F%E6%8E%A5%E5%85%A5
